Heber/src/main/java/springApp/java/com/example/demo/exceptions/GlobalException.java

```diff
@@ -0,0 +1,23 @@
+package springApp.java.com.example.demo.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalException {
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException ex) {
+        return new ResponseEntity<>("Tipo de instrumento no válido", HttpStatus.BAD_REQUEST);
+    }
```

Comment:
No te olvides lo quehablamos, podemos encadenar excepciones al mismo "catch" como lo hacemos en el Try, separando por "," las clases:

```suggestion
    @ExceptionHandler(IllegalArgumentException.class, Exception.class)
    public ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException ex) {
        return new ResponseEntity<>("Tipo de instrumento no válido", HttpStatus.BAD_REQUEST);
    }
```